cfgov/jinja2/v1/_includes/atoms/textarea.html

```diff
@@ -0,0 +1,49 @@
+{# ==========================================================================
+
+   form.textarea()
+
+   ==========================================================================
+
+   Description: Builds form textarea field.
+
+   label:       Name of the field.
+
+   helperText:  String for helper text. Defaults to false.
+
+   required:    Whether the field is required. Defaults to false.
+
+   disabled:    Whether the field is disabled. Defaults to false.
+
+   ========================================================================== #}
+
+{% if not value %}
+    {% set lipsumText = lipsum(1, false, 10, 25) | safe %}
+    {% do global_dict.update({
+        'textarea': {
+            'label':        'This is a label',
+            'helperText':   'This is helper text',
+            'required':     true,
+            'disabled':     false
+        }
+    }) %}
+    {% set value = global_dict.textarea %}
+{% endif %}
+
+{% from 'macros/util/format/url.html' import slugify as slugify %}
+{% set id_slug = slugify( value.label ) %}
+<div class="form-l_col form-l_col-1">
+    <label for="form_{{ id_slug }}" class="form-label-header">
+        {{ value.label }}
+        {%- if value.required == false -%}
+            <span class="micro-copy">&nbsp;(Optional)</span>
+        {%- endif -%}
+    </label>
+    {%- if value.helperText -%}
+        <p class="micro-copy" id="helpertext_{{id_slug}}">{{ value.helperText | safe }}</p>
+    {%- endif -%}
+    <textarea class="input__long"
+              id="form_{{ id_slug }}"
+              {{ 'required' if value.required else '' }}
+              {{ 'disabled' if value.disabled else '' }}
+              {{ 'aria-describedby="helpertext_{{id_slug}}"' if value.helperText else '' }}></textarea>
```

Comment:
I suspect `{{id_slug}}` in `'aria-describedby="helpertext_{{id_slug}}"'` will not be parsed because of the quotes. Please double-check.